src/js/components/Notification/__tests__/Notification-test.tsx

```diff
@@ -63,30 +64,28 @@ describe('Notification', () => {
     expect(onClose).toBeCalled();
   });
 
-  [
+  const validPositions: LayerPositionType[] = [
     'top',
     'bottom',
     'left',
     'right',
-    'start',
-    'end',
     'center',
     'top-left',
     'top-right',
     'bottom-left',
     'bottom-right',
-  ].forEach((positions) =>
-    test(`position ${positions}`, () => {
+  ];
+
+  validPositions.forEach((position) =>
+    test(`position ${position}`, () => {
       render(
         <Grommet>
           <Notification
             id="position-test"
-            toast={{ position: positions }}
+            toast={{ position }}
             title="title"
-            message="message"
-          >
-            This is a layer
-          </Notification>
```

Comment:
I am not sure if this is the intended use case of Notification, as no children props are being destructured and used.

Reply:
It looks like Notification currently isn't expecting anything to be passed through as children. Can we change this test to just remove the `This is a layer` part?

Reply:
yes I will change it to:

```javascript
<Notification
    id="position-test"
    toast={{ position }}
    title="title"
    message="message"
/>
```